### DISCUSSION TITLE
Adding file path on Synology

### DISCUSSION THREAD

**Discussion by zensphinx** [photoview/photoview#204] [Q&A and Troubleshooting] [answered]
Hi! Many thanks for your work. I have installed photoview via docker on my synology NAS and it is running well. My NAS contains a folder with all my pictures (/volume1/backup/pictures) but I cannot add this path so photoview can see it. I get this message: Server error: "invalid root path at (userAddRootPath)". I would be grateful for any guidance on how to solve this. Many thanks1

**Comment by viktorstrate** [comment, marked as answer]
Hi @zensphinx thank you!
Because it's running inside a Docker container, you have to write the path from _within_ the container and not the path on the host machine. You have to mount the photos to a path inside the conatiner using Docker volumes. In your `docker-compose.yml` file under volumes make sure your path is mounted properly, something like this

```yaml
    volumes:
      - api_cache:/app/cache

      # Change this to the directory where your photos are located on your server.
      # If the photos are located at `/home/user/photos`, then change this value
      # to the following: `/home/user/photos:/photos:ro`.
      # You can mount multiple paths, if your photos are spread across multiple directories.
      - /volume1/backup/pictures:/photos:ro
```

This means that the path `/volume1/backup/pictures` on the host machine, will be mounted to the path `/photos` inside the container. After making this change and restarting the container, you should be able to set the root path to `/photos` from the web interface.

**Reply by zensphinx** [reply]
Many thanks! It is working now.
